backend/app/services/ai_agent_bridge.py

```
@@ -0,0 +1,431 @@
+
+    async def _connect_and_stream(self) -> None:
+        """Relay /ai-agent/stream 에 연결해 ai_decision 이벤트를 읽는다."""
+        client = self._get_client()
+        # Relay 는 ai_decision 이벤트를 공용 /sensors/stream SSE 에 포함 발행한다
+        # (기존 프론트 useSensorData 와 동일 스트림 구독, event 필터링으로 분리)
+        url = "/api/v1/sensors/stream"
+
+        async with client.stream("GET", url, timeout=_SSE_READ_TIMEOUT) as resp:
+            if resp.status_code == 404:
+                logger.info("ai_agent_bridge.sse_not_supported (Relay stream unavailable)")
+                raise RuntimeError("SSE endpoint not found (404)")
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)